src/static/clientConfig.ts

```diff
@@ -71,6 +74,9 @@ export default class ClientConfig<Params extends BaseUriParameters>
     if (config.proxy) {
       this.proxy = config.proxy;
     }
+    if (config.throwOnBadResponse !== undefined) {
+      this.throwOnBadResponse = config.throwOnBadResponse;
+    }
```

Comment:
```suggestion
this.throwOnBadResponse = !!config.throwOnBadResponse;
```
Ensures that the property is always a boolean.